racker/salus-telemetry-presence-monitor
# Fluency config needs to be conditionally imported

# What

While randomly testing apps I noticed it was trying to ping fluentd even without `production` profile activated.

# How

Wraps the import in a conditional like we do in the other apps. This one was the only spot missing this conditional.

## How to test

Start up the app and make sure there aren't logs like this:

```
o.k.f.f.i.sender.heartbeat.Heartbeater   : ping(): failed, config=Config{host='localhost', port=24224, intervalMillis=1000}
```
